storage/sealer/manager_test.go

```diff
@@ -239,6 +242,8 @@ func TestSnapDeals(t *testing.T) {
 	ss, err := proofType.SectorSize()
 	require.NoError(t, err)
 
+	getGrothParamFileAndVerifyingKeys(ss)
```

Comment:
AFAIC tell, this was previously working because the sealer_test.go's `TestDownloadParams` ([link](https://github.com/filecoin-project/lotus/blob/327760acffd8b34cc1ad9281e1a16564f76933f3/storage/sealer/ffiwrapper/sealer_test.go#L274)) reliably ran before these tests, ensuring the params existed. That is no longer happenening with go1.20 for reasons that aren't clear to me. This seems like a decent fix.

Reply:
Could just be
```
paramfetch.GetParams(ctx, build.ParametersJSON(), build.SrsJSON(), ss)
```